src/server/ua_server_async.c

```diff
@@ -431,11 +532,14 @@ UA_Server_processServiceOperationsAsync(UA_Server *server, UA_Session *session,
 
     /* Finish / dispatch the operations. This may allocate a new AsyncResponse internally */
     uintptr_t respOp = (uintptr_t)*respPos;
-    uintptr_t reqOp = *(uintptr_t*)((uintptr_t)requestOperations + sizeof(size_t));
+    //uintptr_t reqOp = *(uintptr_t*)((uintptr_t)requestOperations + sizeof(size_t));
```

Comment:
Should the commented-out code be included?

Reply:
This change is intended. The commented out line will be removed in the next commit. In the past there was no requests parameter and the function was only used for method calls. With this in mind, the pointer to methodsToCallSize within the UA_CallRequest was handed in. Adding the size of size_t pointed then to the *methodsToCall. With the new requests parameter this step is not needed any more.